packages/api/src/helpers/getApyFromTo.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  Address,
+  descale,
+  scale,
+  SECONDS_IN_YEAR,
+} from '@voltz-protocol/commons-v2';
+import { pullClosestIndices } from '@voltz-protocol/commons-v2';
+import { ethers } from 'ethers';
+
+async function getLiquidityIndexAt(
+  chainId: number,
+  rateOracle: Address,
+  targetTimestamp: number,
+): Promise<number | null> {
+  const closestIndices = await pullClosestIndices(
+    chainId,
+    rateOracle,
+    targetTimestamp,
+  );
+
+  if (!closestIndices || closestIndices?.length != 2) {
+    return null;
+  }
+
+  const [before, after] = closestIndices.sort(
+    (a, b) => a.blockTimestamp - b.blockTimestamp,
+  );
+
+  const scaleWad = scale(18);
+  const descaleWad = descale(18);
+
+  const fromToIndexDelta = scaleWad(after.liquidityIndex).sub(
```

Comment:
why do we load the calculation with scaling and descaling? We can use just raw JS numbers and calculations.

(i.e. do the calculation directly `(after.liquidityIndex - before.liquidityIndex) * (after.blockTimestamp - before.blockTimestamp) / (targetTimestamp - before.blockTimestamp) +  before.liquidityIndex`)

Reply:
While testing, I noticed the difference `after.liquidityIndex - before.liquidityIndex` could be very small and it would overflow/underflow. I think it's safer to have scaling